### PR TITLE
Add lakhan345's file

### PR DESCRIPTION
**Pull request by lakhan345**
i am new here

### CODE REVIEW

File: test/test-markdown-external.html [hunk 1]
```diff
@@ -18,20 +18,34 @@
 		<div class="reveal" style="display: none;">
 
 			<div class="slides">
-				<!-- <section data-markdown="simple.md" data-separator="^\r?\n\r?\n\r?\n" data-separator-vertical="^\r?\n\r?\n"></section> -->
+				<section data-markdown="simple.md" data-separator="^\r?\n\r?\n\r?\n" data-separator-vertical="^\r?\n\r?\n"></section>
```
Comment (Mershmelow) [inline test/test-markdown-external.html]: test/test-iframe-backgrounds.html

**Review by drumenbassik — APPROVED** [review]

**Review by drumenbassik — APPROVED** [review]

**Comment by mel311** [comment]
Good ppl ..... frame work . :) lost

File: test/test-markdown-external.html (hunk 1 above)
Comment (mel311) [reply test/test-markdown-external.html]: Test/test - frame-backgrounds.url

File: css/reveal.scss [hunk 2]
```diff
@@ -1,9 +1,9 @@
-/*!
+/**
  * reveal.js
  * http://revealjs.com
  * MIT licensed
  *
```
Comment (Arman7717) [inline css/reveal.scss:5]: Hu

**Pull request closed without merging**